docs/how-to/pin-files.md

```diff
@@ -40,3 +44,9 @@ ipfs pin rm -r <foo hash>
 ipfs repo gc
 ipfs cat <foo hash>
 ```
+
+## Local vs. remote pinning
```

Comment:
```suggestion
## Local versus remote pinning
```

Trying to avoid abbreviations as much as possible, they might cause issues with translation efforts down the road.